vsctl/command/cluster.go

```diff
@@ -34,15 +34,17 @@ import (
 )
 
 var clusterVersionList = []string{
-	"v0.5.0", "v0.5.1", "v0.5.2", "v0.5.3", "v0.5.4", "v0.5.5", "v0.5.6", "v0.5.7", "v0.6.0",
+	"v0.5.0", "v0.5.1", "v0.5.2", "v0.5.3", "v0.5.4", "v0.5.5", "v0.5.6", "v0.5.7", "v0.6.0", "v0.7.0",
```

Comment:
remove the old version?

1. vsctl v0.7.0 not support vanus v0.6.0
2. only retain stable and reliable version